OneSignalSDK/onesignal/build.gradle

```diff
@@ -1,24 +1,23 @@
 apply plugin: 'com.android.library'
 
 android {
-    compileSdkVersion 27
-    buildToolsVersion '27.0.3'
-
+    compileSdkVersion 28
     defaultConfig {
+        manifestPlaceholders = [onesignal_app_id: 'b2f7f966-d8cc-11e4-bed1-df8f05be55ba',
```

Comment:
Let's check to make sure by adding the setting of these `manifestPlaceholders` that it doesn't hardcode them into the generated .aar

Reply:
didnt mean to push this honestly, will take it out

Reply:
You are right, it gets sync errors from Android Studio if we remove these. Odd thing is that `gradlew onesignal:assembleRelease` works just fine.

However if we keep these static values generated `.aar` also includes them instead of keeping the placeholders.

I found a work around though, if we redefine the placeholders it fixes the sync error & correctly keeps the placeholders in the build .aar file. See https://github.com/OneSignal/OneSignal-Android-SDK/pull/856/commits/1e93337ad05342233f59c5b3f49dffa04ccaa1cb